site/_includes/posts/sso/starting.md

```diff
@@ -0,0 +1,35 @@
+Once the FusionAuth service is started, open a browser and access [http://localhost:9011/](http://localhost:9011/). If you are not using Docker to run the FusionAuth application, you first have to complete the instructions from the **"Maintenance Mode"** section below before running these steps. If you are using Docker, you'll be taken to the **"FusionAuth Setup Wizard"**, as seen in the image below.
+
+{% include _image.liquid src="/assets/img/blogs/includes/fusionauth/setup-wizard.png" alt="FusionAuth setup wizard." class="img-fluid" figure=true %}
+
+Fill in the admin user account details, read and accept the License Agreement. When you are ready, click **"Submit"**.
+
+{% include _callout-tip.liquid content="Learn more about the [Setup Wizard here](https://fusionauth.io/docs/v1/tech/tutorials/setup-wizard)." %}
+
+After submitting, you'll be taken to a login screen where you need to fill in the credentials you specified during the setup wizard and sign in to the FusionAuth administrative user interface. Later, you'll use the same admin account for testing the SSO of the application.
+
+#### Maintenance Mode
```

Comment:
I want to have folks stay away from maintenance mode for tutorials. Let's stick with docker.

Reply:
You can rollback the changes to the django post, let's leave that one be.